airflow/contrib/operators/ecs_operator.py

```diff
@@ -96,6 +111,10 @@ def __init__(self, task_definition, cluster, overrides,
         self.platform_version = platform_version
         self.network_configuration = network_configuration
 
+        self.awslogs_group = awslogs_group
+        self.awslogs_region = awslogs_region
```

Comment:
The region is always going to be the same for logs isn't it, so we don't need a separate region here do we? (we should reuse `region_name` I think?)

Reply:
The CloudWatch region can actually be different from the region of your ECS cluster. If you leave the `self.awslogs_region` empty it will use your default region.

Reply:
You could also argue that if `awslogs_region` is empty is should default to `region_name` instead of the default region. What do you think?

Reply:
Might it be clearer to use them in this order: `awslogs_region`, `region_name`, region from connection?

Reply:
Yep makes sense. I'll make the change, thanks.

Reply:
Comments crossed in flight :) Could you update the docs too?